com.unity.mobile.notifications/Tests/Runtime/Android/AndroidNotificationTests.cs

```diff
@@ -10,6 +10,7 @@ class AndroidNotificationTests
     private static int receivedNotificationCount = 0;
 
     [Test]
+    [UnityPlatform(RuntimePlatform.Android)]
```

Comment:
I think you can have this attribute on top of class, no need to apply it for every test